### PR TITLE
Improve PR template with instructions for title

### PR DESCRIPTION
**Pull request by FokkeZB**
@hansemannn

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-**JIRA**: https://jira.appcelerator.org/browse/TIMOB-[TICKET-NUMBER]
+**JIRA:** https://jira.appcelerator.org/browse/[TICKET]
 
-**Additional description (optional)**: 
+Provide a clear PR title prefixed with `[TICKET]`
```
Comment (hansemannn) [inline .github/PULL_REQUEST_TEMPLATE.md]: That's already part of the guidelines, the PR template itself should be as template-like as possible.

Comment (FokkeZB) [inline .github/PULL_REQUEST_TEMPLATE.md]: I know, but many won't read ;)

Comment (hansemannn) [inline .github/PULL_REQUEST_TEMPLATE.md]: Ok. But let's wire up a bet: I swear that we will receive a PR the next week that begins with literally `[TICKET]` :-)

**Comment by hansemannn** [comment]
Approved!

**Pull request merged**